sw/device/lib/dif/dif_otbn.c

```diff
@@ -201,33 +217,28 @@ dif_otbn_result_t dif_otbn_is_busy(const dif_otbn_t *otbn, bool *busy) {
   return kDifOtbnOk;
 }
 
-dif_otbn_result_t dif_otbn_get_err_code(const dif_otbn_t *otbn,
-                                        dif_otbn_err_code_t *err_code) {
-  if (otbn == NULL || err_code == NULL) {
+static const uint32_t kDifOtbnErrBitsAll = (1 << kDifOtbnErrBitBadDataAddr) |
```

Comment:
I'm not sure this is the best way to go about this. In particular if we add a new error bit there's nothing that will immediately break here so we'd incorrect view the new bit as an undefined one. Sadly reggen doesn't generate a mask value that tells us all defined valid bits as far as I can tell which could be used here along with an extra static assert checking the OR of all the bits matches the mask.